### PR TITLE
src: Handle WEBHOOK_UPDATE events

### PR DESCRIPTION
**Pull request by KingDGrizzle**
**Please describe the changes this PR makes and why it should be merged:**

This PR adds handling of the WEBHOOK_UPDATE event from Discord.
Note that the data returned is only the guild ID and channel ID.

**Semantic versioning classification:**  
- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/client/websocket/packets/handlers/WebhookUpdate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const AbstractHandler = require('./AbstractHandler');
+const { Events } = require('../../../../util/Constants');
+
+class WebhookUpdate extends AbstractHandler {
+  handle(packet) {
+    const client = this.packetManager.client;
+    const data = packet.d;
+    const guild = client.guilds.get(data.guild_id);
+    const channel = client.channels.get(data.channel_id);
+    if (guild && channel) client.emit(Events.WEBHOOK_UPDATE, guild, channel);
```
Comment (SpaceEEC) [inline src/client/websocket/packets/handlers/WebhookUpdate.js]: `WEBHOOKS_UPDATE`

File: src/util/Constants.js [hunk 2]
```diff
@@ -250,6 +250,7 @@ exports.Events = {
   VOICE_BROADCAST_UNSUBSCRIBE: 'unsubscribe',
   TYPING_START: 'typingStart',
   TYPING_STOP: 'typingStop',
+  WEBHOOK_UPDATE: 'webhookUpdate',
```
Comment (SpaceEEC) [inline src/util/Constants.js]: `WEBHOOKS_UPDATE`

File: src/util/Constants.js [hunk 3]
```diff
@@ -291,6 +292,7 @@ exports.Events = {
  * * VOICE_STATE_UPDATE
  * * TYPING_START
  * * VOICE_SERVER_UPDATE
+ * * WEBHOOK_UPDATE
```
Comment (SpaceEEC) [inline src/util/Constants.js]: `WEBHOOKS_UPDATE`

File: src/util/Constants.js [hunk 4]
```diff
@@ -325,6 +327,7 @@ exports.WSEvents = keyMirror([
   'VOICE_STATE_UPDATE',
   'TYPING_START',
   'VOICE_SERVER_UPDATE',
+  'WEBHOOK_UPDATE',
```
Comment (SpaceEEC) [inline src/util/Constants.js]: `WEBHOOKS_UPDATE`

File: src/client/websocket/packets/WebSocketPacketManager.js [hunk 5]
```diff
@@ -47,6 +47,7 @@ class WebSocketPacketManager {
     this.register(WSEvents.MESSAGE_REACTION_ADD, require('./handlers/MessageReactionAdd'));
     this.register(WSEvents.MESSAGE_REACTION_REMOVE, require('./handlers/MessageReactionRemove'));
     this.register(WSEvents.MESSAGE_REACTION_REMOVE_ALL, require('./handlers/MessageReactionRemoveAll'));
+    this.register(WSEvents.WEBHOOK_UPDATE, require('./handlers/WebhookUpdate'));
```
Comment (SpaceEEC) [inline src/client/websocket/packets/WebSocketPacketManager.js]: `WEBHOOKS_UPDATE`

File: src/client/websocket/packets/handlers/WebhookUpdate.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  }
+}
+
+/**
+ * Emitted whenever a guild channel has it's webhooks changed.
```
Comment (SpaceEEC) [inline src/client/websocket/packets/handlers/WebhookUpdate.js]: `its` (belonging to), not `it's` (it has / it was)

File: src/client/websocket/packets/handlers/WebhookUpdate.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+ * @event Client#webhookUpdate
+ * @param {Guild} guild The guild that had a webhook update
+ * @param {GuildChannel} channel The channel that had a webhook update
```
Comment (SpaceEEC) [inline src/client/websocket/packets/handlers/WebhookUpdate.js]: Webhooks can only be in `TextChannel`s as far as I know.

**Comment by KingDGrizzle** [comment]
I will do typings when I get home :+1:

**Pull request merged**